src/index.js

```diff
@@ -9,13 +9,20 @@ const IS_WINDOWS = platform() === 'win32';
 
 // Helper functions
 const noop = () => null;
-const matches = (key, importee) => {
+const matches = (key, importee, isRegEx) => {
   if (importee.length < key.length) {
     return false;
   }
   if (importee === key) {
     return true;
   }
+
+  if (isRegEx) {
+    if (key.test(importee) === true) {
+      return true;
+    }
```

Comment:
If we have a regular expression, also none of the code later makes any sense. How about just

```js
if (key instanceof RegExp) {
  return key.test(importee)
}
```

?  Also I guess there is no need to compare with `true` as `.test` should always return a boolean.

Reply:
The plan is (was) to also support simple strings. Agreed with the needless if tought. Fixed.